pkgs/applications/version-management/git-and-tools/git-big-picture/default.nix

```diff
@@ -1,23 +1,19 @@
-{ fetchFromGitHub, python3Packages, lib, stdenv, git, graphviz }:
+{ fetchPypi, python3Packages, lib, stdenv, git, graphviz, cram, pytest }:
 
 python3Packages.buildPythonApplication rec {
   pname = "git-big-picture";
-  version = "1.0.0";
+  version = "1.1.1";
 
-  src = fetchFromGitHub {
-    owner = "git-big-picture";
-    repo = pname;
-    rev = "v${version}";
-    sha256 = "14yf71iwgk78nw8w0bpijsnnl4vg3bvxsw3vvypxmbrc1nh0bdha";
+  src = fetchPypi {
+    inherit pname version;
+    format = "wheel";
+    sha256 = "a20a480057ced1585c4c38497d27a5012f12dd29697313f0bb8fa6ddbb5c17d8";
   };
 
   buildInputs = [ git graphviz ];
 
-  # NOTE: Tests are disabled due to unpackaged test dependency "Scruf".
-  #       When bumping to 1.1.0, please re-enable and use:
-  #checkInputs = [ cram git pytest ];
-  #checkPhase = "pytest test.py";
-  doCheck = false;
+  checkInputs = [ cram git graphviz pytest ];
+  checkPhase = "pytest test.py && cram test.cram";
```

Comment:
```suggestion
  checkPhase = ''
    pytest test.py
    cram test.cram
  '';
```

Reply:
@SuperSandro2000 is someone doing `set -e` for me behind the scenes? Where do I find the style guide that you're following here with regard to formatting?

Reply:
> is someone doing set -e for me behind the scenes?

yes.

> Where do I find the style guide that you're following here with regard to formatting?

https://github.com/NixOS/nixpkgs/blob/master/doc/contributing/coding-conventions.xml

Reply:
Then please take my word for it. This is how it is usually done and how it should be done here.